pandas/__init__.py

```diff
@@ -170,8 +170,8 @@
 from pandas.io.json import _json_normalize as json_normalize
 
 from pandas.util._tester import test
-import pandas.testing
-import pandas.arrays
+from pandas import testing
+from pandas import arrays
```

Comment:
Is there a reason why `api`, `testing`, and `arrays` was not imported with "from pandas import"? I changed it to `from pandas import` to include them in `__all__`.

Reply:
these should be equivalent (ex the name space visiblity).

note that we may not be fully testing this (see in pandas/tests/api/test_api.py)

Reply:
I added a test for `__all__` and found in that process a few un-exported modules.